prodtests/sim_challenge.sh

```diff
@@ -124,6 +124,11 @@ engine="TGeant3"
 # options to pass to every workflow
 gloOpt=" -b --run "
 
+# option to set the number of sim workers
+simWorker=""
+
+# option to set the number of tpc-lanes
+tpcLanes=""
```

Comment:
I would for now not make these things a dedicated and required command line options without a default value.
Would it be possible to user environment variables instead? Something like if TPCLANES is set use it, otherwise do nothing?

Side note: I am afraid there will be many more options (such as number of threads for TRD and others). Ideally we should have some algorithm that determines the number of lanes and threads and order of execution just based on the number of CPUS as well as the memory available.

Reply:
Hi Sandro,
just a minor comment.
sim_challenge is run on the grid via a executable script. In the executable script we can manage the check on the CPU resources and pass the right number to sim_challenge independently of what set in the jdl (if we think it is the proper thing to do). I would avoid such a check inside the sim_challenge script.
Cheers
Francesco